plugins/notifications-backend/src/service/router.ts

```diff
@@ -47,7 +56,12 @@ export async function createRouter(
   });
 
   router.get('/notifications', (request, response, next) => {
-    const { pageSize, pageNumber } = request.query;
+    const {
+      pageSize,
+      pageNumber,
+      orderBy = 'created',
```

Comment:
I think defaults should not be here. should be in handler. reason 1 is to have router without the handler logic. reason 2 is because we may have more callers to the handler function an they will expect default as well. E.g. tests.